.gitignore

```diff
@@ -2,3 +2,4 @@
 *.phar
 .phpunit.result.cache
 docker-compose.override.yml
+.php_cs.cache
```

Comment:
Totally unsure about this: I would prefer that in a global gitignore, but I'm fine merging it anyway. @antonmedv @peterjaap if you have any opinion on that, tell me :)

Reply:
I think it’s okay.